cmd/atlas/atlas.go

```diff
@@ -99,21 +99,35 @@ func createConfigFromMongoCLIConfig() {
 	_, err = io.Copy(out, in)
 	if err != nil {
 		log.Printf("There was an error generating %s: %v", atlasConfigPath, err)
+		return
 	}
 
 	_, _ = fmt.Fprintf(os.Stderr, `AtlasCLI has copied your MongoCLI configuration to: %s
 
 `, atlasConfigPath)
 }
 
-func mongoCLIConfigPath() string {
-	configDir, err := config.MongoCLIConfigHome()
-	if err != nil {
-		return ""
+func mongoCLIConfigFilePath() (path string, err error) {
+	configPath := ""
+	configDir := ""
+
+	if configDir, err = config.MongoCLIConfigHome(); err == nil {
+		configPath = fmt.Sprintf("%s/config.toml", configDir)
 	}
 
-	configPath := fmt.Sprintf("%s/mongocli.toml", configDir)
-	return configPath
+	// Check if file exists, if any error is detected try to get older file
+	if _, err = os.Stat(configPath); err == nil {
+		return configPath, nil
+	}
+
+	if configDir, err = config.OldMongoCLIConfigHome(); err == nil {
+		configPath = fmt.Sprintf("%s/mongocli.toml", configDir)
+	}
+
+	if _, err = os.Stat(configPath); err == nil {
+		return configPath, nil
+	}
+	return "", err
```

Comment:
```suggestion
	return os.Stat(configPath)
```

Reply:
this returns FileInfo object and we're returning file path here? 
```
type FileInfo interface {
	Name() string       // base name of the file
	Size() int64        // length in bytes for regular files; system-dependent for others
	Mode() FileMode     // file mode bits
	ModTime() time.Time // modification time
	IsDir() bool        // abbreviation for Mode().IsDir()
	Sys() interface{}   // underlying data source (can return nil)
}
```

Reply:
true, my bad